templates/InstallChocolatey.ps1.erb

```diff
@@ -21,7 +21,7 @@ $ErrorActionPreference = 'Stop'
 #try {
 
 # variables
-$url = '<%= @download_url %>'
+$url = '<%= @download_url %><% if @chocolatey_version.to_s =~ /\d+\./ %>chocolatey.<%= @chocolatey_version %>.nupkg<% end %>'
```

Comment:
The documentation states that the download url should download a nupkg if you go to it on a bare url. So this change would not be enough.

Reply:
Moreover it would be an incorrect change (to clarify).